src/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function promiseSerial(tasks: Array<() => Promise<any>>) {
```

Comment:
What 's the purpose of this file?

Reply:
This is where commonly-used, generic functions will go.

Reply:
The first function I've added (`promiseSerial`) executes a lists of async tasks in series.

Reply:
ACK!